pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -36,6 +36,12 @@
         "win-x64": {
             "#import": [ "win" ]
         },
+        "win-arm": {
```

Comment:
These were absent before because initially 'win' was meant to indicate `vista` but we didn't want to use the `vista` name.  Since then we've re-purposed `win` to indicate a version-less OS family, so I'm OK having arm RIDs.